src/test/spec/read_write_concern/operation.rs

```diff
@@ -0,0 +1,12 @@
+use tokio::sync::RwLockReadGuard;
+
+use crate::test::{run_spec_test, LOCK};
+
+use super::super::run_v2_test;
+
+#[cfg_attr(feature = "tokio-runtime", tokio::test)]
+#[cfg_attr(feature = "async-std-runtime", async_std::test)]
+async fn run() {
+    let _lock: RwLockReadGuard<_> = LOCK.run_concurrently().await;
```

Comment:
Ah I think I know what the problem is. The v2 runner executes `killAllSessions` between test executions, and because this is running concurrently with other tests that are using implicit sessions, their operations get terminated and result in that error. So if we update this to be an exclusive lock, we should be good.

Reply:
ahh okay good catch, fixed